### ISSUE TITLE
Can not login with Stack Exchange OpenID

### ISSUE THREAD

**Issue by MoriTanosuke** [JabbR/JabbR#424]
When I try to log in with my own domain and _Stack Exchange Open ID_ as delegate, I get this error message:

```
[InvalidOperationException: 'Carsten Ringe' is not a valid user name.]
   JabbR.Services.ChatService.AddUser(String userName, String identity, String email) in c:\dev\github\JabbR\JabbR\Services\ChatService.cs:287
   JabbR.Auth.Login.ProcessRequest(HttpContext context) in c:\dev\github\JabbR\JabbR\Auth\Login.ashx.cs:93
   System.Web.CallHandlerExecutionStep.System.Web.HttpApplication.IExecutionStep.Execute() +625
   System.Web.HttpApplication.ExecuteStep(IExecutionStep step, Boolean& completedSynchronously) +270
```

**Comment by davidfowl** [comment]
It's because of spaces in your name. We could autofix up these types of logins.

**Comment by samandmoore** [comment]
This was fixed in PR #432
